client/components/faq/FaqHeader.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
+import PropTypes from 'proptypes';
+import { FAQS } from '@components/common/FaqContent';
+import Search from '@assets/faq/search-outline.svg';
+
+const FaqHeader = ({
+  updateSearch,
+  updateCount,
+  updateResults,
+}) => {
+  const [searchInput, updateSearchInput] = useState('');
+  const faqSearchAutoComplete = FAQS.map(item => item.question.toLowerCase());
+  const autocompleteDiv = document.querySelector('.autocomplete-items');
+  const searchBox = document.querySelector('#autocompete-search');
```

Comment:
I'm pretty sure querySelector is causing the two errors you mentioned. Try useRef instead of document.querySelector.

https://reactjs.org/docs/hooks-reference.html#useref

Reply:
That worked perfectly, took care of the errors!